### ISSUE TITLE
Add rendering for amenity=taxi

### ISSUE THREAD

**Issue by kocio-pl** [gravitystorm/openstreetmap-carto#879]
I think amenity=taxi should be visible as an icon - with name, if it has one - when it's a point. An area tagged like this could be (maybe) rendered like parking area but with taxi icon instead of "P".

**Comment by kocio-pl** [comment]
We miss taxi icon - @nebulon42, could you try to draw one?

**Comment by pnorman** [comment]
Taxi is a tough icon to do at small sizes in a language/culture independent manner. Most taxis are the same shape as cars here

**Comment by daganzdaanda** [comment]
I think the word TAXI is very international. It's also used on maps already as an icon.
I've tried putting the 4 letters in a 16px square.

https://gist.github.com/daganzdaanda/6c195a64265e6c5e177c

![taxi-rounded-16](https://cloud.githubusercontent.com/assets/7693297/6158737/c7848ca0-b24d-11e4-8579-474161981773.png)
![taxi-16](https://cloud.githubusercontent.com/assets/7693297/6158738/c7861822-b24d-11e4-98fd-558fd4bc27a9.png)
![taxi-cube-16](https://cloud.githubusercontent.com/assets/7693297/6158739/c793d89a-b24d-11e4-8b66-0d46e90ee9db.png)

![taxi-3x](https://cloud.githubusercontent.com/assets/7693297/6158745/d7deefdc-b24d-11e4-829b-71d36dcf2c63.png)

**Comment by HolgerJeromin** [comment]
if only the name Taxi i would suggest to to use the full height.
is a front view of a car with a sign on top possible with 16 px?

**Comment by math1985** [comment]
osm-icons has http://osm-icons.org/images/d/df/12257.svg.

Something like http://www.barcelona-airport.com/images/icons/taxi.png would work for me, but I don't know how internationally recognizable this is.

**Comment by kocio-pl** [comment]
+1 - I was thinking exactly about something like that.

**Comment by mkoniecz** [comment]
> Something like http://www.barcelona-airport.com/images/icons/taxi.png would work for me, but I don't know how internationally recognizable this is.

It is probably looks exactly like normal car at 16px or 20px.

**Comment by kocio-pl** [comment]
There's an iconic font (https://github.com/FortAwesome/Font-Awesome) with scaled taxi picture to check different sizes:

http://fontawesome.io/icon/taxi/

I think even if it looks more like a normal car, we just have to make sure it differs enough from other car icons we use (like car rental, car repair station or something).

The second issue is a background color - we can make it more/the same as in car parking, since it looks alike (cars staying most of the time). As @daganzdaanda said, taxi is like a bus stop, but while the people are usually waiting for buses and the platform is the most important thing, taxi cabs are usually waiting for people, hence their staying place is more visible and the "platform" for passangers can be neglected most of the time.

**Comment by nebulon42** [comment]
I've create an issue, so I don't forget.
As TAXI is quite international it is hard to get into a 16px square. It will be difficult to have it different form other car icons, but it is worth a try.

**Comment by nebulon42** [comment]
Icon is ready for some time, still a car: ![taxi-16](https://cloud.githubusercontent.com/assets/3531092/7732767/71c4e18c-ff2a-11e4-97fa-f386a2645e57.png)

**Comment by math1985** [comment]
Looks good. Icon is recognizable by random person.

**Comment by kocio-pl** [comment]
+1 and +1 =}

**Comment by govvin** [comment]
Could this icon be re-considered? The "iconic" taxi cab is a western thing, while in many parts of the world, they come in many forms, sizes, and range (tuktuks, rickshaws, moto-taxis, pedicabs).

**Comment by matthijsmelissen** [comment]
> Could this icon be re-considered?

We can always reconsider icons. Do you have a proposal?

**Comment by govvin** [comment]
Hi @matthijsmelissen . I've been looking at icons/pictograms at the Noun project, and found interesting ones, but I don't I found anything "universal" enough.  

The earlier [suggestion](https://github.com/gravitystorm/openstreetmap-carto/issues/879#issuecomment-73987702) by daganzdaanda to use "taxi" looks like a good starting point, especially the first one - short and sweet:
![](https://cloud.githubusercontent.com/assets/7693297/6158745/d7deefdc-b24d-11e4-829b-71d36dcf2c63.png)

**Comment by HolgerJeromin** [comment]
Please open a new issue for that. Thanks

**Comment by HolgerJeromin** [comment]
Reopening has the downside that there is much irrevant text to read.
We often have the problem that the discussion is hard to follow with too much input

**Comment by Adamant36** [comment]
That can be an issue, but then you also lose positive parts of the issue like the original icon proposals and why they where rejected. I guess there's pros and cons of either way of doing it. 

Have you looked into a Github app like FastHub? I use it sometimes. It still takes some scrolling for longer issues, but its easier to follow along with things then just doing it in mobile Chrome.

**Comment by kocio-pl** [comment]
This one is completely other issue. One can link to original comment if needed.
